content/doc/book/installing/war-file.adoc

```diff
@@ -38,10 +38,10 @@ The Jenkins Web application ARchive (WAR) file can be started from the command l
 * Unlike downloading and running Jenkins with Blue Ocean in Docker
   (<<docker,above>>), this process does not automatically install the Blue Ocean
   features, which would need to installed separately via the
-  link:../../book/managing[**Manage Jenkins**] >
-  link:../../book/managing/plugins/[**Manage Plugins**] page in Jenkins. Read
+  link:../../../book/managing[**Manage Jenkins**] >
+  link:../../../book/managing/plugins/[**Manage Plugins**] page in Jenkins. Read
   more about the specifics for installing Blue Ocean on the
-  link:/doc/book/blueocean/getting-started/[Getting started with Blue Ocean]
+  link:../../../book/blueocean/getting-started/[Getting started with Blue Ocean]
```

Comment:
Make absolute? Will then be easier to find when reorganizing stuff, and won't break when moving something from `index.adoc` to `whatever.adoc`. (Same applies to other files.)